simple_salesforce/login.py

```diff
@@ -12,7 +12,7 @@
 import requests
 
 
-def SalesforceLogin(**kwargs):
+def SalesforceLogin(session=None, **kwargs):
```

Comment:
> Based on the convention below I've been doing kwargs.get('session') instead of defining it inside the actual definition line.

Yeah, I'm generally not a fan of that for two reasons: 

1. It doesn't really make sense if keywords aren't arbitrary (i.e. I can pass in `foo` as a kwarg because there's no validation done.
2. It takes away from immediately readability (you actually /have/ to go through the docs or implementation to figure them out). That also applies to generated docs (also on my TODO list).

> But mixing both doesn't make a ton of sense.

Fair point. I wanted to minimize the surface area of the change, but I think you're right and moving to named kwargs for consistency makes the most sense.

> Also you'll probably want to add something to the docstring about this.

Woops, thanks for the catch.